server/sonar-web/src/main/js/components/issue/views/comment-form-view.js

```diff
@@ -82,7 +82,7 @@ export default PopupView.extend({
     const text = this.ui.textarea.val();
     const update = this.model && this.model.has('key');
     const method = update ? 'edit_comment' : 'add_comment';
-    const url = '/api/issues/' + method;
+    const url = window.baseUrl + '/api/issues/' + method;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Convert this concatenation to the use of a template. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=javascript%3AS3512)